openwhisk/zip_test.go

```diff
@@ -54,3 +55,23 @@ func Example_jar() {
 	// ./action/unzip/exec.jar
 	// <nil>
 }
+
+func Example_venv() {
```

Comment:
Indeed, this is a great addition for the venv use cases and quite unfortunate that the go zip packages do not have a built in means to turn this on and have made devs. walk/handle reading/writing of these cases themselves for years apparently.  

It makes me wonder if there is a need to review other langs. like Ruby that also have venvs widely adopted using similar mechanisms as python.

Reply:
I think the usual way we do the virtualenv embedding does not use symbolic links, but I introduced the ability to generate a virtualenv using the image itself, and this creates symbolic links that refers to what is in the image itself... and zip code choked on it. So I had to fix. The fix is generic we can do the same also for ruby or deno or other actionloop based runtimes